docs/api/search.md

```diff
@@ -0,0 +1,107 @@
+Search
+======
+
+####Our api supports searching artifact metadata which will return a link header for each search result.
```

Comment:
Could you use "----" instead?  Thats what we usually use.  I also prefer the look when viewing it in text